webservices/common/util.py

```diff
@@ -3,9 +3,11 @@
 import flask
 import ujson
 
+from datetime import datetime
 
 dirname = os.path.dirname
 MAIN_DIRECTORY = dirname(dirname(dirname(__file__)))
+CURRENT_CYCLE = datetime.now().year + datetime.now().year % 2
```

Comment:
I think we already have a `get_cycle` utility method defined somewhere - [in the config](https://github.com/18F/openFEC/blob/develop/webservices/config.py#L9-L21) (cycle start and cycle end, plus a `CURRENT_YEAR` constant).  Could/should those be leveraged instead, or is it more useful to have it handled this way now?  I just want to make sure we're not introducing duplicate logic. :-)

Reply:
I think this is more straightforward since it is a simple formula, but maybe I am biased to my own preferences. I can change it Monday if you have a strong opinion about it.